### PR TITLE
209-initial-content-version

### PR DESCRIPTION
**Pull request by yishain11**

### CODE REVIEW

File: frontend/src/components/TextEditorPopup.tsx [hunk 1]
```diff
@@ -102,7 +102,7 @@ const TextEditorPopup = (props: DialogProps) => {
         >
           <ContentEditor
             content={newTextContent}
-            placeholder={t(StringBank.INSERT_NEW_VERSION)}
+            placeholder={newTextContent ? newTextContent.text : t(StringBank.INSERT_NEW_VERSION)}
```
Comment (avirankatz) [inline frontend/src/components/TextEditorPopup.tsx:105]:
```suggestion
            placeholder={t(StringBank.INSERT_NEW_VERSION)}
```

**Review by avirankatz — CHANGES_REQUESTED** [review]
Thanks! Exactly what's needed.
Please see minor requests

File: frontend/src/components/TextEditorPopup.tsx [hunk 2]
```diff
@@ -26,11 +26,14 @@ interface DialogProps {
   completeBtnText: string;
   cancelBtnText: string;
   variabels?: Record<string, unknown>;
+  initialContent: JSONContent | undefined;
```
[the remark this answers is not in the document]
Comment (yishain11) [reply frontend/src/components/TextEditorPopup.tsx:30]: done

File: frontend/src/components/TextEditorPopup.tsx (hunk 1 above)
Comment (avirankatz) [reply frontend/src/components/TextEditorPopup.tsx:105]: It is the content, just not the placeholder

Comment (yishain11) [reply frontend/src/components/TextEditorPopup.tsx:105]: but the placeholder is what is shown.
anyway, I changed as requested

**Review by avirankatz — APPROVED** [review]

**Pull request merged**